src/BookStore/Application/Command/AnonymizeBooksCommandHandler.php

```diff
@@ -22,8 +22,6 @@ public function __invoke(AnonymizeBooksCommand $command): void
             $book->update(
                 author: new Author($command->anonymizedName),
             );
-
-            $this->bookRepository->save($book);
```

Comment:
It's not necessary normally, as the book instance comes from the repository and is already persisted at this point

Reply:
hmm I find it good for the sake of explicitness though; also doctrine is an infrastructure detail here (does it work with the memory adapter btw?)

Reply:
Yes, it does. I will update the tests to avoid the transaction middleware which is not necessary for testing mode.